definitions/reports/grouping.rb

```diff
@@ -0,0 +1,23 @@
+module Checks
+  module Report
+    class Grouping < ForemanMaintain::Report
+      metadata do
+        description 'Check how resources are grouped'
+      end
+
+      def run
+        collection_count = sql_count('SELECT COUNT(*) FROM katello_host_collections')
+        collection_count_with_limit = sql_count("SELECT COUNT(*) FROM katello_host_collections WHERE unlimited_hosts = 'f'")
+        hostgroup = sql_count('SELECT COUNT(*) FROM hostgroups')
+        hostgroup_nest_level = sql_count("SELECT COALESCE(MAX((CHAR_LENGTH(ancestry) - CHAR_LENGTH(REPLACE(ancestry, '/', '')))), 0) + 1 AS count FROM hostgroups")
+        table_preference_count = sql_count('SELECT COUNT(*) FROM table_preferences')
```

Comment:
This counts if anyone ever set it, it will wrongly count even cases where someone explicitly set the selection to the default value. Do we need to worry about that case?

Reply:
Can we somehow filter the default value? Is it hardcoded anywhere? It is probably not a big deal, depends on how complicated it would be...

Reply:
Unless I'm overlooking something it is defined only in application code. We would also need to consider multiple defaults depending on for which table the preferences are set in case we ever get that functionality for other tables as well